doc/api/child_process.md

```diff
@@ -1671,6 +1707,16 @@ within the child process to close the IPC channel as well.
 The `subprocess.exitCode` property indicates the exit code of the child process.
 If the child process is still running, the field will be `null`.
 
+When the child process is terminated by a signal, `subprocess.exitCode` will be
+`null`. To get the corresponding POSIX exit code, use
+[`util.convertProcessSignalToExitCode(subprocess.signalCode)`][`util.convertProcessSignalToExitCode()`].
+
+**Note:** The `child_process` module does not automatically set `exitCode` when
+a process is terminated by a signal to avoid breaking changes in existing code
+that may depend on `exitCode` being `null` in such cases. The
+[`util.convertProcessSignalToExitCode()`][] utility function is provided to
+allow applications to opt-in to this conversion when needed.
```

Comment:
I'd drop this paragraph. Saying "we don't change this API because changing it would be a breaking change" applies to virtually _any_ API we have. Providing a single link to `util.convertProcessSignalToExitCode()` should be sufficient (it's not something that people are going to need a lot). (EDIT: and maybe it makes more sense to link to that from the `.signalCode` docs than from the `.exitCode` docs?)